### ISSUE TITLE
Airlike node wielditems with no tiles inherit surrounding node textures and can be animated (!)

### ISSUE THREAD

**Issue by GreenXenith** [minetest/minetest#8228]
This is one of the weirdest and awesomest bugs ive seen in a while.
Create a node without tiles and the `airlike` drawtype:
```lua
minetest.register_node("test:animated", {
	description = "animated",
	drawtype = "airlike",
})
```

**Short video demonstrating various oddities: https://youtu.be/JtNPsQIbvv0**

Noticed behaviors:
* It will change sizes/shapes based on last wielded item (hold item, switch to the airlike node)
  * hand will make it 2x1
  * Mesh will make it tiny
  * Node/nodebox/anything else will make it normal
  * Something makes it 1x2, haven't figured out what it was yet
* Inherits surrounding nodes
* Not in video: Inherits node texture on punch for about a second
  * In that brief period yaw will change the selected texture (if the node has multiple)
* Not in video: Works on any drawtype*, but it will always be a cube (which means it may look funky)
* *it does not work on butterflies or fireflies
* Changes based on yaw (or extreme pitch, up or down)
* **Can be animated**
* Sometimes applies one texture to all sides
* Upon entering a new chunk where there are no textures it defaults to a font atlas

Discovered by @LoneWolfHT, tested more by me.

__Two things:__
What the heck :)
and wielded items can be animated; this needs to be an actual feature.

**Comment by LoneWolfHT** [comment]
Animated water sources look great as a wielditem too

**Comment by rubenwardy** [comment]
The fuck

**Comment by DS-Minetest** [comment]
Maybe related: #7304

--------------------------------------

>and wielded items can be animated; this needs to be an actual feature.

:+1: 
A little bit related: #4758

---------------------------------------

Imo this is not a bug. The modder is responsible to give the item a good look and airlike nodes being completely invisible in inventory and wieldhand would be worse. Ergo this is even a bugfix for the bug of not being able to distinguish between airlike node items and "no item". (I don't think that the bug will reproduce enough to cause a plague.)
A note in the documentation would probably be good.

**Comment by appgurueu** [comment]
This deserves the "obscure" tag

**Comment by Wuzzy2** [comment]
This was fixed a while ago in this PR: #9299
It can be closed now.
